src/localization/locales/en-US.json

```diff
@@ -585,7 +607,8 @@
         "validation_field_max_length": "$t({{path}}) must be fewer than {{max}} characters.",
         "validation_array_min": "$t({{path}}) must have at least {{min}} item.",
         "validation_number_integer": "$t({{path}}) must be an integer.",
-        "validation_number_max": "$t({{path}}) must be less than {{max, number()}}."
+        "validation_number_max": "$t({{path}}) must be less than {{max, number()}}.",
+        "helper_text_info_label": "[TODO] Help for: $t({{label}})"
```

Comment:
question: how is this used?

Reply:
@paulschreiber This is the aria-label for the Info button that is just an icon, so it needs an aria-label